lua/default-config.lua

```diff
@@ -200,7 +200,36 @@ O = {
       },
       filetypes = { "php", "phtml" },
     },
-    latex = {},
+    latex = {
```

Comment:
Could you please review these options again ?
Check
https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua
https://github.com/latex-lsp/texlab/blob/master/docs/options.md

I didn't check everything , but I know for sure some of the options don't exist (aux_directory, bibtex_formatter)
Also, please keep a consistent order, put the single line entries on top (The one of lspconfig is great, please follow it)

Reply:
Ok. I will follow the lspconfig with the only destinction that the single line entries on top. The settings auxDirectory and bibtexFormatter also exist in the lspconfig. Only in the table they are called O.lang.latex.aux_directory and O.lang.latex.bibtex_formatter.
I looked at other settings and thought this might be a kind of naming convention.